src/main.py

```diff
@@ -20,13 +20,20 @@ def sub(a: int, b: int) -> Result[int, Type[NumberTooSmallException]]:
     else:
         return Ok(a - b)
 
+def mul(a: int, b: int) -> Result[int, Type[NumberTooSmallException]]:
```

Comment:
NO! i dont like `mul`
I want it to be multiply
```suggestion
def multiply(a: int, b: int) -> Result[int, Type[NumberTooSmallException]]:
```